### PR TITLE
Feature/1.6/queue/cli message attribute extension

### PR DESCRIPTION
**Pull request by lazan**
I think this is a pretty "easy" commit to review if you ignore ws to get rid of a few ns-colapse.

As always, review the commit message, which is far more important than information here. This PR will "disappear", but the git log will stay for ever... 

It's the commit 3c8661f83176ec8cb96ebd92901f22daa8c05f2c that is of interest.

### CODE REVIEW

File: middleware/administration/unittest/source/cli/test_queue.cpp [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+            EXPECT_TRUE( std::regex_match( output, std::regex{ R"(.*properties: foo,.*\n)"})) << CASUAL_NAMED_VALUE( output); 
+         }
+
+         // available
+         {
+            auto output = local::execute( R"(echo "casual" | casual buffer --compose | casual queue --attribues available 42s | casual pipe --human-sink)");
+            EXPECT_TRUE( std::regex_match( output, std::regex{ R"(.*available: 42.*\n)"})) << CASUAL_NAMED_VALUE( output); 
+         }
+         
+         //constexpr std::string_view command =
```
Comment (aliothe) [inline middleware/administration/unittest/source/cli/test_queue.cpp]: remove

**Comment by lazan** [comment]
Ok, the PR culture/method seems to be lacking somewhat 😄 

`main reviewer: ` @AlbsAgren, the rest is optional.

File: middleware/administration/source/cli.cpp [hunk 2]
```diff
@@ -262,19 +265,65 @@ Note: only works for 'servers' with a message pump)"
...
+
+                     return common::argument::Option{
+                        std::move( invoke),
+                        { "--human-sink"},
+                        R"(INCUBATION - serialize casual pipe messages to human readable form, and in practice sink the message 
+
+@attention INCUBATION - might change during, or in between minor version.
+
+@attention: This is one way, there is no possibility to go from the output 
+   back to _real_ pipe messages. Use only for stuff that are supposed to be discared.)"
```
Comment (AlbsAgren) [inline middleware/administration/source/cli.cpp]: Nitpick: spelling error in "discared"

File: middleware/queue/source/manager/admin/cli.cpp [hunk 3]
```diff
@@ -1101,7 +1121,102 @@ casual queue --peek <queue-name> <id1> <id2> | <some other part of casual-pipe>
...
+                        return local::queues();
+
+                      return { "<value>"};
+                  };
+
+                  return argument::Option{
+                     argument::option::one::many( std::move( invoke)),
+                     std::move( complete),
+                     { "--attribues"},
+                     R"(INCUBATION - adds or mutates queue message attribues on piped messages
```
Comment (AlbsAgren) [inline middleware/queue/source/manager/admin/cli.cpp]: Nitpick: "attribues"

**Review by AlbsAgren — DISMISSED** [review]

**Review by AlbsAgren — APPROVED** [review]

**Comment by lazan** [comment]
I don't understand why github thinks that this branch has conflicts, and at the same time says that it's up to date with the target branch... 

Manually merged and pushed...

**Pull request closed without merging**